aws/elasticache/elasticache.tf

```
@@ -0,0 +1,44 @@
+###
+# AWS Elasticache Redis for Notification application
+###
+
+resource "aws_elasticache_subnet_group" "notification-canada-ca-cache-subnet" {
+    name       = "notification-canada-ca-${var.env}-cache-subnet"
+    subnet_ids = var.vpc_private_subnets
+}
+
+resource "aws_elasticache_cluster" "notification-cluster-cache" {
+    # Default is false with this param, it looks counter-intuitive because
+    # applied changes would only happen during maintenance window if false.
+    apply_immediately    = true
+    cluster_id           = "notification-cluster-cache"
```

```suggestion
    cluster_id           = "notification-canada-ca-${var.env}-cluster-cache"
```
I think we put the env in all our names